Titanium.Web.Proxy/ProxyServer.cs

```diff
@@ -348,6 +353,12 @@ public void Start()
                 certificateCacheManager.TrustRootCertificate();
             }
 
+            if (ForwardToUpstreamGateway && GetCustomUpStreamHttpProxyFunc == null && GetCustomUpStreamHttpsProxyFunc == null)

```

Comment:
ForwardToUpstreamGateway property works just like that. You have to set it true to get PAC/WAPD script handling working; by default it does not chains to PAC script configuration.